src/cunumeric/unary/unary_red_util.h

```diff
@@ -91,6 +97,39 @@ struct UnaryRedOp {
   static constexpr bool valid = false;
 };
 
+template <legate::LegateTypeCode TYPE_CODE>
+struct UnaryRedOp<UnaryRedCode::ALL, TYPE_CODE> {
+  static constexpr bool valid = true;
+
+  using VAL = legate::legate_type_of<TYPE_CODE>;
+  using OP  = Legion::ProdReduction<VAL>;
+
+  template <bool EXCLUSIVE>
+  __CUDA_HD__ static void fold(VAL& rhs1, VAL rhs2)
+  {
+    OP::template fold<EXCLUSIVE>(rhs1, rhs2);
+  }
+};
+
+template <>
+struct UnaryRedOp<UnaryRedCode::ALL, legate::LegateTypeCode::COMPLEX128_LT> {
+  static constexpr bool valid = false;
+};
```

Comment:
Was this necessary? I see that we have this on prior operators, but it looks like it wasn't necessary for ANY right below.

Reply:
It seems to be required for this operation since it uses PROD Reduction. 
I get following error if I remove it:
```
cunumeric/unary/unary_red_omp.cc:167:47:   required from here
./cunumeric/unary/unary_red_template.inl:50:88: error: no matching function for call to ‘legate::Store::reduce_accessor<cunumeric::UnaryRedOp<cunumeric::UnaryRedCode::ALL, COMPLEX128_LT>::OP, (cunumeric::VariantKind::OMP != cunumeric::VariantKind::GPU), 2>(Realm::Rect<2, long long int>&) const’
   50 |     auto lhs = args.lhs.reduce_accessor<typename OP::OP, KIND != VariantKind::GPU, DIM>(rect);
```